src/System.Security.Cryptography.Algorithms/src/System/Security/Cryptography/RSA.cs

```diff
@@ -20,6 +20,16 @@ public abstract partial class RSA : AsymmetricAlgorithm
         protected abstract byte[] HashData(byte[] data, int offset, int count, HashAlgorithmName hashAlgorithm);
         protected abstract byte[] HashData(Stream data, HashAlgorithmName hashAlgorithm);
 
+        public virtual byte[] DecryptValue(byte[] rgb) 
+        {
+           throw new NotSupportedException(); // Same as Desktop
+        }
+
+        public virtual byte[] EncryptValue(byte[] rgb) 
+        {
+           throw new NotSupportedException(); // Same as Desktop
```

Comment:
These seem to be missing the resource strings?  And seem to be indented 3 spaces, instead of 4?